viewpoint_generation_node:
  ros__parameters:
    visualize: True
    cuda_enabled: False

    model:
      mesh:
        file: "package://viewpoint_generation/assets/cube_beveled.stl"
        units: 'in'
      point_cloud:
        file: ""
        units: 'in'
        sampling:
          ppsqmm: 4.
          number_of_points: 100000
    
    curvature:
      number_of_neighbors: 30

    camera:
      fov:
        width: 0.02
        height: 0.03
      dof: 0.02
      focal_distance: 0.35

    regions:
      region_growth: False
